### PR TITLE
Turbo Isolation Valve Logic 

### PR DESCRIPTION
**Pull request by divyak1910**
- Updated Turbo Isolation valve logic

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU [hunk 1]
```diff
@@ -66,11 +66,21 @@ fb_VP_VVC_4(
 );
 
 // Turbo Isolation valve - Interlocked with turbo pump operation
-fb_PP_VRC_4(
-    i_xExtILK_OK := FALSE,
-    i_xOverrideMode := xSystemOverrideMode_BTS
-);
-fb_PP_VRC_4.M_Set_OPN_SW(value:= (fb_PP_PMF.q_stPump.eState = E_PumpState.pumpRUNNING) );
+IF (fb_PP_PMF.q_stPump.eState <> E_PumpState.pumpRUNNING) THEN		// IF Roughing Pump is NOT stopped
```
Comment (slactjohnson) [inline lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU:69]: The comment needs to be updated to reflect the logic

File: lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU [hunk 2]
```diff
@@ -66,11 +66,21 @@ fb_VP_VVC_4(
...
-    i_xOverrideMode := xSystemOverrideMode_BTS
-);
-fb_PP_VRC_4.M_Set_OPN_SW(value:= (fb_PP_PMF.q_stPump.eState = E_PumpState.pumpRUNNING) );
+IF (fb_PP_PMF.q_stPump.eState <> E_PumpState.pumpRUNNING) THEN		// IF Roughing Pump is NOT stopped 
+	fb_PP_VRC_4.M_Set_OPN_SW(FALSE);
+ELSE
+	fb_PP_VRC_4.M_Set_OPN_SW(TRUE);
+END_IF
+
+IF (fb_PP_PMF.q_stPump.eState = E_PumpState.pumpSTOPPING) THEN
```
Comment (slactjohnson) [inline lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU:75]: Have you examined the pump state machine to make sure that we will enter this state in an off-normal event, e.g. an error?

File: lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU [hunk 3]
```diff
@@ -66,11 +66,21 @@ fb_VP_VVC_4(
...
+	fb_PP_VGC.M_Set_OPN_SW(FALSE);
+END_IF
+
+//fb_PP_VRC_4(
```
Comment (slactjohnson) [inline lcls-plc-las-bts/las_bts/POUs/PRG_VALVES.TcPOU:79]: Why is this commented out? If it's not needed, should it be removed?

**Review by slactjohnson — CHANGES_REQUESTED** [review]
A few comments on the code. I'm still not comfortable merging this until we've resolved the TMC file issue.

**Comment by divyak1910** [comment]
Older branch , New pull request created with all updates

**Pull request closed without merging**